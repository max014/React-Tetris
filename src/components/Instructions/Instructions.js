import React from 'react';
import styles from './Instructions.module.css';

export const Instructions = (props) => (
	<div className={styles.Instructions}>
		<h4>Rotate</h4>
		<div className={styles.Container}>
			<div className={styles.Key}>
				S
			</div>
			<div className={styles.Key}>
				D
			</div>
		</div>
	</div>
);

export default Instructions;