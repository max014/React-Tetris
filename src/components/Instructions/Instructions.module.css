.Instructions{
	margin: 25px 0;
	border: 4px solid white;
	background: #666;
}

.Key{
	background-color: #111;
	display: inline-block;
	width: 60px;
	height: 60px;
	color: white;
	text-align: center;
	border-radius: 5px;
	margin-bottom: 10px;
	padding-top: 17px;
	box-sizing: border-box;
	font-size: 20px;
}

.Instructions h4{
	color: white;
	text-align: center;
	margin: 10px 0;
}

.Container{
	display: flex;
	justify-content: space-around;
}