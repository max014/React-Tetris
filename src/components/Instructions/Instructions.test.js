import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Instructions } from './Instructions';

describe('Instructions', () => {
    const instructions = shallow(<Instructions/>);
    it('renders properly', () => {
        expect(toJson(instructions)).toMatchSnapshot();
    });
});