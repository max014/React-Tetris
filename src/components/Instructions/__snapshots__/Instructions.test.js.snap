// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions renders properly 1`] = `
<div
  className="Instructions"
>
  <h4>
    Rotate
  </h4>
  <div
    className="Container"
  >
    <div
      className="Key"
    >
      S
    </div>
    <div
      className="Key"
    >
      D
    </div>
  </div>
</div>
`;
